portfolio/src/main/webapp/style.css

```diff
@@ -45,6 +45,15 @@ button {
   font-size: 15px;
 }
 
+input {
+  background-color: black;
+  color: white;
+  border: none;
+  padding: 15px 32px;
+  display: inline-block;
+  font-size: 15px;
+}
+
 a {
```

Comment:
what is a? can be addressed in a separate PR.

Reply:
a is for the links

Reply:
can you add a comment?